tools/prettyJSON.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python2
```

Comment:
Agree. The code I added ; validates the file is a standard name, checks the json itself is valid, and outputs the results to a _new.json version of the file. I'll drop the commit.